eng/pipelines/runtime.yml

```diff
@@ -385,6 +385,25 @@ jobs:
     jobParameters:
       liveRuntimeBuildConfig: release
 
+#
+# Build libraries using live CoreLib
+# These are part of the test coreclr generic test build.
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
+    buildConfig: Release
```

Comment:
@sdmaclea do we need to build libraries in Release here? We already build it on Debug: https://github.com/dotnet/runtime/blob/master/eng/pipelines/runtime.yml#L382

Do we need the release build?

Also, this will introduce a duplicate job of the libraries on rolling builds because there's already a libraries job for Release in OSX, because https://github.com/dotnet/runtime/blob/master/eng/pipelines/runtime.yml#L382 will be built on release. See, `variables.debugOnPrReleaseOnRolling`.

@trylek this might be the reason of the break

Reply:
I agree this is definitely at least part of the problem, thanks for pointing that out. I'm investigating whether it's the only one.

Reply:
The duplicate job was prevented by the conditional logic below
```yaml
      condition: >-
        and(
          ne(variables['debugOnPrReleaseOnRolling'], 'Release'),
          or(
            eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true),
            eq(variables['isFullMatrix'], true)))
```